src/csharp/Grpc.IntegrationTesting/Payloads.cs

```diff
@@ -0,0 +1,580 @@
+// Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
For now, generated C# code gets checked into the repo. Ideally, we wouldn't do this, but it is not so simple to make this work with visual studio projects in a way that wouldn't lead to some visual studio awkwardness. So checking in generated code was the intention here.